docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -270,6 +271,8 @@ The table displays each Exception list on an individual row, with the most recen
 
 TIP: To view details of the rule the Exception list is assigned to, click the link in the `Rules assigned to` column.
 
+To filter Exception lists by a specific value, enter key words or a KQL query in the Search bar.
```

Comment:
It's technically not a KQL query - it's similar to Github where you can search `author:xyz`. The attributes you can specify are `name`, `list_id`, `created_by`, `type`, `tags`. So to search by who created it and name it would be `name:"My List" created_by:username`

Reply:
If no attribute is specified, it searches by list name by default.